corehq/blobs/migrate.py

```diff
@@ -500,9 +539,10 @@ class gmigrator:
                 print("done.")
 
 
-MIGRATIONS = {m.slug: m for m in [
-    BackendMigrator("migrate_backend", BlobDbBackendMigrator),
-    BackendMigrator("migrate_backend_check", BlobDbBackendCheckMigrator),
+MIGRATIONS = {m().slug: m for m in [
+    lambda: BackendMigrator("migrate_backend", BlobDbBackendMigrator),
+    lambda: BackendMigrator("migrate_backend_check", BlobDbBackendCheckMigrator),
+    lambda: BackendMigrator("compress_form_xml", BlobDBCompressionMigrator, type_code=CODES.form_xml),
```

Comment:
I don't see why these were converted to lambdas. Can you explain that?

Reply:
If they are statically created then changing any fields on the objects persist for the duration of the runtime. This isn't an issue in a management command but in test cases if there are multiple calls to the same migration then the field values from previous tests persist.